src/App.js

```diff
@@ -30,7 +29,9 @@ const App = () => {
   const changeTaskStatus = (taskID) => {
     const updated = todoTasks.map((todo) => {
       if (todo.id === taskID) {
+        /* eslint-disable no-param-reassign */
         todo.done = !todo.done;
+        /* eslint-enable no-param-reassign */
```

Comment:
yes I know but for Microverse Eslint roles they never give access to reassign object value so instead of making a change in eslintric files I just ignored it in this line :+1: :+1: